docs/routing-rules.md

```diff
@@ -50,15 +50,30 @@ If there is error parsing the routing rules configuration file, an error is
 logged, and requests are routed using the routing group header
 `X-Trino-Routing-Group` as default.
 
+### Configuring API Requests with HttpClient Config
+
+Trino Gateway uses the Airlift HttpClient library when making HTTP requests.
```

Comment:
Why do we want to mention the library and the class names?

Reply:
I thought it might not be obvious, so would be better off to tell them that you need to check Airlift conf, which is slightly different than other http clients

Reply:
There is no need to mention such names because they are not important in my opinion. Just linking to https://trino.io/docs/current/admin/properties-http-client.html looks sufficient.

Reply:
Sounds fair enough :)